stock_picking_invoice_link/i18n/es.po

```diff
@@ -101,5 +101,22 @@ msgstr "Transferir"
 msgid "You can not modify an invoiced stock move"
 msgstr "No puede modificar un movimiento de almacén facturado"
 
+<<<<<<< HEAD
```

Comment:
This conflict must be solved

Reply:
That's what I wanted to say. My memory is limited :smile: